templates/content_navbar.hbs

```diff
@@ -45,25 +45,19 @@
         <div class="navbar-start">
             <div class="navbar-item has-dropdown is-hoverable">
                 <!-- show the active project name -->
-                {{#if (active_project request.spin-full-url "/spin/" )}}
+                {{#if (active_project request.spin-full-url "/cloud/" )}}
                     <a class="navbar-link is-active is-hoverable">
-                        <span>Spin</span>
+                        <span>Cloud</span>
                     </a>
                 {{else}}
-                    {{#if (active_project request.spin-full-url "/cloud/" )}}
+                    {{#if (active_project request.spin-full-url "/wasm-functions/" )}}
                         <a class="navbar-link is-active is-hoverable">
-                            <span>Cloud</span>
+                            <span>Wasm Functions</span>
                         </a>
```

Comment:
Do we want to present `cloud` as the top option in the drop down. I do not have strong feelings about this but just wanted to check. oh, I also see it just keeps the same structure as before.

Reply:
Right, same order remains; just removed Spin.